backend/requirements.txt

```diff
@@ -14,12 +14,13 @@ pyasn1==0.4.2
 pyasn1-modules==0.2.1
 PyJWT==1.5.2
 PyYAML==3.12
-requests==2.10.0
+requests==2.3
```

Comment:
why downgrade?

Reply:
fcm library wasn't compatible to that version, I had to change to the 2.3 version. I think none of our services has been affected by this change.